src/main/java/core/basesyntax/service/writer/WriteFileFruitImpl.java

```
@@ -0,0 +1,16 @@
+package core.basesyntax.service.writer;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteFileFruitImpl implements WriteFileFruit {
+    @Override
+    public void writeAll(String string, String path) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(path))) {
+            bufferedWriter.write(string);
+        } catch (IOException e) {
+            System.out.println("error: file not found." + e);

```

This catch block prints the exception instead of rethrowing. This violates the checklist item: "Don't ignore exceptions in try-catch blocks". Per the checklist you should rethrow a RuntimeException with an informative message and include the original exception as the cause, for example:

```
} catch (IOException e) {
    throw new RuntimeException("Can't write file: " + path, e);
}
```

Replace the System.out.println usage with such a rethrow so callers can react to the failure and tests can assert expected error conditions.